Resources/Prototypes/_RMC14/Roles/Jobs/Other/Fax_Responders/clf_responder.yml

```
@@ -0,0 +1,95 @@
+- type: job
+  parent: CMJobBase
+  id: RMCCLFResponder
```

Job and PlaytimeTracker prototypes should start with CM instead of RMC until we can convert them all
```suggestion
  id: RMCCLFResponder
```